x/gov/msgs/deposit_msg.go

```diff
@@ -0,0 +1,60 @@
+package gov
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type DepositMsg struct {
+	ProposalId int64          `json:"proposal_id"` // ID of the proposal
+	Depositer  crypto.address `json:"depositer"`   // Address of the depositer
```

Comment:
Can we remove this and let it be determined according to whoever signed the tx? 

Currently this is unauthenticated. Or we could leave it and just make sure to check it against the Signer. Note the Spec does not include a Depositer in this message type: https://github.com/cosmos/cosmos-sdk/blob/develop/docs/spec/governance/transactions.md#deposit

Reply:
It is from this that the tx knows who signed it.  The tx only has the signatures, and then it calls `GetSigners()` on the Msg.  In this DepositMsg, `GetSigners()` returns `Depositer`.